RoddyCore/test/de/dkfz/roddy/plugins/LibrariesFactoryTest.groovy

```diff
@@ -139,17 +146,80 @@ public class LibrariesFactoryTest {
         assert mapOfAvailablePlugins["B"]["1.0.3-0"]?.previousInChain == mapOfAvailablePlugins["B"]["1.0.2-2"] && mapOfAvailablePlugins["B"]["1.0.3-0"]?.previousInChainConnectionType == PluginInfo.PluginInfoConnection.EXTENSION
     }
 
-    public static PluginInfoMap callLoadMapOfAvailablePlugins(List<File> additionalPluginDirectories = []) {
-        List<File> pluginDirectories = [new File(Roddy.getApplicationDirectory(), "/dist/plugins/")]
-        pluginDirectories += additionalPluginDirectories
+    @BeforeClass
+    public static void setupTestDataForPluginLoaderTest() {
```

Comment:
Setting up all those directories for a test is a lot of effort. Can we change the code such that we get two functions, one that does all the filesystem checks and another that takes a list of directories (that are already checked for existence and readability) and process that? The first function (plus tests) we may then move to the LocalFilesystemAccessProvider (where it belongs; alternatively RuntimeService, that also does some FS tests in the moment) while the latter can be tested without any FS access.